cli/commands/patchMigrations.js

```diff
@@ -0,0 +1,73 @@
+var Promise = require('bluebird');
+
+var fs = Promise.promisifyAll(require('fs'));
+
+let frameworkName = 'Transmute Framework'
+
+let patchBegin = `🦄 ${frameworkName}`
+let patchEnd = `🐩 ${frameworkName}`
```

Comment:
If people are going to be forced to use a hacky ass cli they might as well get to see some emojis.